app/views/tutorials/show.html.erb

```diff
@@ -66,6 +67,7 @@
     <%= @facade.current_video.description.truncate(50) %>...
     <%= link_to "More", "#", "data-action" => "click->tutorials#showDescription", id: @facade.current_video.id, class: "show-link"%>
   </p>
+  <% end %>
 </div>
 
 </main>
```

Comment:
Future iteration -- we should probably add a link to add a video to the tutorial on this page somewhere.